.vscode/settings.json

```
@@ -1,16 +1,12 @@
 {
```

I didn't realize that these themes were pre-existing. I'll revert them.
```suggestion
{
  "workbench.colorCustomizations": {
    "activityBar.background": "#22285C",
    "titleBar.activeBackground": "#303981",
    "titleBar.activeForeground": "#F9FAFD"
  },
```